models/Form.js

```diff
@@ -0,0 +1,53 @@
+const adult = require('./Adult.js').schema;
```

Comment:
🚫 **[eslint]** <[import/extensions](https://github.com/import-js/eslint-plugin-import/blob/v2.25.3/docs/rules/extensions.md)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Unexpected use of file extension "js" for "./Adult.js"